CHANGES.md

```diff
@@ -1,3 +1,29 @@
+## v2.9.2 (2022-01-18)
+[View detailed changelog](https://github.com/ivadomed/ivadomed/compare/v2.9.1...release)
+
+**FEATURE**
+
+- Implementation of Random Blur Augmentation.  [View pull request](https://github.com/ivadomed/ivadomed/pull/1034)
+- Implementation of Random Bias Field Augmentation.  [View pull request](https://github.com/ivadomed/ivadomed/pull/1033)
+- Implementation of Random Gamma Contrast Augmentation.  [View pull request](https://github.com/ivadomed/ivadomed/pull/1015)
```

Comment:
The changelog looks reasonable to me! 

But, at what point do we go to `2.10.0` rather than `2.9.2`? I'm not sure how major these features are, but they are features!

Reply:
2.10 will most likely be made once multi-session, mutli-contrasts support goes in as that is a major feature under planning for a while.

Reply:
If that's a major feature, does it warrant 3.0.0? `MAJOR.MINOR.PATCH`, etc.

(I also recognize, though, that I haven't really been present to ivadomed meetings, so it's not really my place to comment, either!)